config/environments/production.rb

```diff
@@ -28,7 +28,7 @@
   # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = false
+  config.assets.compile = true
```

Comment:
for this PR we shouldn't need to change the source code. we can remove this change